src/main/resources/test_files/runtime-spark/RumbleML/AnnotateFunction/MLAnnotateForChaining10.jq.disabled

```diff
@@ -0,0 +1,11 @@
+(:JIQS: ShouldRun; Output="" :)
+let $data := structured-json-file("./src/main/resources/queries/rumbleML/sample-ml-data-sentence.json")
+
+let $tokenizer := get-transformer("Tokenizer")
+let $intermediate-data := $tokenizer($data, {"inputCol": "sentence", "outputCol": "output"})
+let $df-intermediate-data := annotate($intermediate-data, {"id": "integer", "sentence": "string", "output": "[\"string\"]"})
```

Comment:
when we do support it, the syntax will be "output" : [ "string" ] :)

Reply:
Fixed